### DISCUSSION TITLE
Github does not display the code module when logging in to the account.

### DISCUSSION THREAD

**Discussion by zhangwenda0518** [community/community#84908] [Other Feature Feedback, Questions, & Ideas]
My previous use was normal, and this problem began to appear yesterday. I tried to change the network and click on more warehouses. The same problem exists, and I still have no solution directly.
Interestingly, after I logged out of my account, everything returned to normal, but when I logged in again, the problem appeared again.

![image](https://github.com/community/community/assets/77140740/306f0c12-1458-41bf-a45d-9c468cdd1ebe)

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @zhangwenda0518 , we're happy you're here! You are more likely to get a useful response if you are posting your question( in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by DTS-11** [comment]
Hi there!

Have you tried refreshing your browser cache? If not, please try it using the shortcut `Ctrl + f5`

**Comment by Mohammedishaqsadiq** [comment]
you are right

**Comment by Mohammedishaqsadiq** [comment]
i test it that's right
